Automattic/jetpack
# Tiled gallery: rounded corners

Adds rounded corners feature to tiled gallery block.

#### Changes proposed in this Pull Request:
* Rounded corners. 🎉 

<img width="668" alt="Screenshot 2020-02-16 at 23 16 16" src="https://user-images.githubusercontent.com/87168/74613103-29c20b80-5114-11ea-96b2-6d7d54a82c48.png">

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* 💪 _enhance_

#### Testing instructions:
- Add tiled gallery block and some images to it

- Confirm that you can make images nicely round with the sidebar setting. 

    <img width="1096" alt="Screenshot 2020-02-16 at 23 16 57" src="https://user-images.githubusercontent.com/87168/74613099-1dd64980-5114-11ea-8316-ef8a625d637f.png">

- Oh, except with Circle layout which obviously is already pretty darn round.
    <img width="1078" alt="Screenshot 2020-02-16 at 23 18 46" src="https://user-images.githubusercontent.com/87168/74613138-77d70f00-5114-11ea-88ac-a1d4027e2441.png">

- Store galleries without this change and try to open them with this change; confirm nothing breaks.

- Test the roundness at the frontend of the site


#### Proposed changelog entry for your changes:
* Allow adding rounded corners to Tiled gallery images
